features/support/world.js

```diff
@@ -56,7 +56,7 @@ function ensurePipelineExists(config) {
             .then(response => {
                 this.jwt = response.body.token;
 
-                return this.createPipeline(config.repoName, branch);
+                return this.createPipeline(config.repoName, branch, config.rootDir);
```

Comment:
can you add `config.rootDir` to jsdoc too

Reply:
@tkyi Thank you. I added.